objectivec/docs/jazzy_config.yaml

```diff
@@ -0,0 +1,15 @@
+module: ONNX Runtime Objective-C API
```

Comment:
`objectivec/docs` and `objectivec/tests` are copied from onnxruntime.

I think we can simplify the release process by replicating the entire `objectivec` directory.

Reply:
wfm